### PR TITLE
Add django40 support

### PR DESCRIPTION
**Pull request by aht007**

### CODE REVIEW

**Review by UsamaSadiq — APPROVED** [review]

File: .github/workflows/pypi-publish.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install pip
+        run: pip install pip setuptools wheel
```
Comment (mraarif) [inline .github/workflows/pypi-publish.yml:22]: we should use a `pip.txt` to install these

File: setup.py [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
+VERSION = get_version("multi_email_field", "__init__.py")
+
+
 setup(
-    name='django-multi-email-field',
-    version='0.6.2',
-    author='Florent Lebreton',
-    author_email='florent.lebreton@makina-corpus.com',
-    url='https://github.com/fle/django-multi-email-field',
+    name='dj-multi-email-field',
```
Comment (mraarif) [inline setup.py]: have you checked that this name is available on PyPI?

**Review by mraarif — COMMENTED** [review]
make sure you've tested this with test PyPI with twine

File: setup.py (hunk 2 above)
Comment (aht007) [reply setup.py]: yes it is available.

**Comment by aht007** [comment]
Hey @nedbat , We have forked this repo into openedx and before releasing this package I wanted to know if we are good to go and publish this package with the existing [license](https://github.com/openedx/django-multi-email-field/blob/master/LICENSE). Also if you could direct us to any documentation regarding this process.
Thanks

**Comment by nedbat** [comment]
@aht007 I don't quite understand the goal here.  I would have assumed we wanted to contribute these changes back upstream.  But you've removed the authors' names from the README and changed the package name? Why aren't we contributing this back?

**Comment by aht007** [comment]
@nedbat The repo isn't being maintained by the owners and is not in active development for a long time.  We wanted to assign this repo to `Open Craft` so that they can maintain it but they haven't worked on one previous issue `Django-fernet-fields` that was assigned to them two years ago. So instead we have forked this in `openedx` so we can maintain it here as our packages are dependent on it. That is why I was saying that maybe you can direct us to the standard process of publishing the package under `openedx`

**Comment by UsamaSadiq** [comment]
`django-multi-email-field` got a new PyPI release [0.7.0](https://pypi.org/project/django-multi-email-field/0.7.0/) after I opened up an issue in the upstream repo https://github.com/fle/django-multi-email-field/issues/23. We’ve got this Django upgrade blocker resolved now so we can resume our progress on upgrading edx packages without forking the package.

**Pull request closed without merging**